drools-wb-screens/drools-wb-pmml-text-editor/drools-wb-pmml-text-editor-api/src/main/java/org/drools/workbench/screens/pmmltext/type/PMMLResourceTypeDefinition.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.drools.workbench.screens.pmmltext.type;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Default;
+import javax.inject.Inject;
+
+import org.guvnor.common.services.project.categories.Model;
+import org.uberfire.backend.vfs.Path;
+import org.uberfire.workbench.annotations.VisibleAsset;
+import org.uberfire.workbench.category.Category;
+import org.uberfire.workbench.type.ResourceTypeDefinition;
+
+@Default
+@VisibleAsset
+@ApplicationScoped
+public class PMMLResourceTypeDefinition
+        implements ResourceTypeDefinition {
+
+    private Category category;
+
+    public PMMLResourceTypeDefinition() {
+    }
+
+    @Inject
+    public PMMLResourceTypeDefinition(final Model category) {
```

Comment:
Why `Model`? I think `Decision` fit better.

Reply:
This was based on [this](https://issues.jboss.org/browse/DROOLS-4049?focusedCommentId=13735451&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13735451) and subsequent comments.

Reply:
@etirelli @danielezonca I don't mind what category you'd prefer PMML to be in.. but please both agree! "Model" is traditionally Java classes, "Decision" is DMN, DRL etc. "Other" is anything else that we can't categorise more explicitly.

Reply:
Fine for me `Model` too, no need to change :)